app/javascript/pages/country/widget/widgets/widget-tree-gain/widget-tree-gain-actions.js

```diff
@@ -35,6 +44,28 @@ const getTreeGain = createThunkAction(
   }
 );
 
+const getProcessedRankingData = (rawData, params) => {
```

Comment:
This looks like a selector function to me. Lets keep data mapping to the selectors.